### DISCUSSION TITLE
What Can Be Done While Blue Blocker Is Not Approved?

### DISCUSSION THREAD

**Discussion by WindUpCitrus** [kheina-com/Blue-Blocker#317] [General]
Hello,

Blue Blocker was one of my favorite addons. It has done a lot of make my browsing experience on Twitter a lot more tolerable by removing all those spam bots and twitter blues. I have noticed that it has been approximately two months since the last post from the Firefox disapproved the latest update for Blue Blocker to work on the x dot com redirect. What methods could one use in the meantime while we wait patiently for the developer's update to go through? 

I seen that ublock origin's string

twitter.com##article [data-testid="icon-verified"]:upward(article)

Does not seem to work. And other addons such as "Eight Dollars" and "Bluelite blocker" have not been updated in over a year. "Twitter Stress Reduction" seems unable to auto hide them either and "Control Panel For Twitter" doesn't hide the verified main posts. What I have done is use an auto clicker to max block with "one block twitter" to be able to swiftly mass hide posts that I do not care for, but I would prefer a more elegant solution if there is one available.

**Comment by cooljeanius** [comment]
> And other addons such as "Eight Dollars" and "Bluelite blocker" have not been updated in over a year.

Note that for "Eight Dollars" specifically, there's a replacement called "twitter-real-verified": https://github.com/luiserdef/twitter-real-verified

**Reply by WindUpCitrus** [reply]
I supposed but does not help in terms of autoblock. 

I hope the project is not permanently put on ice.

**Comment by cooljeanius** [comment]
Note that v0.4.3 has been released now, so hopefully Mozilla will approve it this time...

**Reply by WindUpCitrus** [reply]
I am ever so glad to hear that. I hope for you and your team's success.
